README.md

```diff
@@ -50,3 +51,9 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+**MY ANSWER:**
+
+The main while loop runs for maxIterationsWithoutImprovement times, which is a constant value. Within this while loop, we have 2 nested loops that iterate over all the elements in the currentRoute array. So, I believe our worst-case asymptotic time complexity should be $\Theta(n^2)$ where n is the number of elements in the currentRoute array.
```

Comment:
But you're resetting whether an improvement has been found when an improvement is found. So the loop doesn't run a constant number of times.